rest-api/migrate-data-from-sendbird-to-talkjs/example-data/messages/example-messages.json

```diff
@@ -0,0 +1,196 @@
+{
+  "messages": [
+    {
+      "type": "MESG",
+      "message_id": 10463242990,
+      "message": "Hi, welcome to the migration channel!",
+      "custom_type": "",
+      "created_at": 1736855618748,
+      "channel_url": "sendbird_group_channel_483424280_ac14e7e8e6d5b3bc2dd3979fb360ad432ae0d6e8",
+      "updated_at": 0,
+      "mentioned_users": [],
+      "mention_type": "users",
+      "is_removed": false,
+      "req_id": "rq-01007c01-7e16-47bf-9350-f0f93c18004e",
+      "data": "",
+      "file": {},
+      "user": {
+        "user_id": "alice-sendbird",
+        "profile_url": "https://talkjs.com/new-web/avatar-6.jpg",
+        "nickname": "Alice",
+        "metadata": {}
+      },
+      "message_survival_seconds": -1,
+      "silent": false,
+      "translations": {}
+    },
+    {
+      "type": "MESG",
+      "message_id": 10463258210,
+      "message": "Hi \ud83d\udc4b",
+      "custom_type": "",
+      "created_at": 1736855722175,
+      "channel_url": "sendbird_group_channel_483424280_ac14e7e8e6d5b3bc2dd3979fb360ad432ae0d6e8",
+      "updated_at": 0,
+      "mentioned_users": [],
+      "mention_type": "users",
+      "is_removed": false,
+      "req_id": "rq-1790448c-47ff-4b05-be99-34cf7fc356ae",
+      "data": "",
+      "file": {},
+      "user": {
+        "user_id": "sebastian-sendbird",
+        "profile_url": "https://talkjs.com/new-web/avatar-2.jpg",
+        "nickname": "Sebastian",
+        "metadata": {}
+      },
+      "message_survival_seconds": -1,
+      "silent": false,
+      "translations": {}
+    },
+    {
+      "type": "MESG",
+      "message_id": 10463264206,
+      "message": "Hi, looking forward to the migration!",
+      "custom_type": "",
+      "created_at": 1736855760789,
+      "channel_url": "sendbird_group_channel_483424280_ac14e7e8e6d5b3bc2dd3979fb360ad432ae0d6e8",
+      "updated_at": 0,
+      "mentioned_users": [],
+      "mention_type": "users",
+      "is_removed": true,
```

Comment:
Love it that you also included a removed message in there, as a nice test :)

Reply:
Ha, that was originally unintentional (I deleted a message while creating my test data in Sendbird), but I thought it would be good to keep in for testing purposes 😀